docroot/profiles/custom/cgov_site/modules/custom/cgov_site_section/cgov_site_section.module

```diff
@@ -38,6 +39,43 @@ function cgov_site_section_entity_bundle_field_info(EntityTypeInterface $entity_
   }
 }
 
+/**
+ * Implements term_cgov_site_sections_parent_validate(), validate default root as parent.
+ */
+function term_cgov_site_sections_parent_validate(&$form, \Drupal\Core\Form\FormStateInterface $form_state) {
+   $langcode =  $form_state->getValue('langcode');
+   $tid =  $form_state->getValue('tid');
+   $vid = 'cgov_site_sections';
+    $terms =\Drupal::entityTypeManager()->getStorage('taxonomy_term')->loadTree($vid);
+    $enParentTermCount = $esParentTermCount = 0;
+    foreach ($terms as $term) {
```

Comment:
Please keep an eye on performance and code efficiency when making the updates @bryanpizzillo mentioned, particularly when using the presave() hook. 

presave() is used recursively when updating certain cgov_site_section fields and loadTree() is expensive in certain circumstances --- we have A LOT of site sections in production so this is potentially nightmare fuel.